### ISSUE TITLE
CalDAV Child parent relationship is not being recognized

### ISSUE THREAD

**Issue by oktayacikalin** [alainm23/planify#1187]
**Describe the bug**
I've a task and some subtasks. One of them is not seen as a child.

**To Reproduce**
I don't know how it came to this state - yet.

**Expected behavior**
All items, which have "RELATED-TO:{{PARENT_ID}}", and do not explicitly say something else, should be seen as children, I guess. At least NextCloud Tasks and aCalendar+ (Android) do show them like this.

**Screenshots**
None

**Desktop (please complete the following information):**
 - Fedora 39, latest GNOME Shell
 - Planify 4.5.2 from flathub

**Additional context**
It looks like the child, which is not seen as it, has a different notation for the relationship. I don't know how it came to this state - yet.

```
BEGIN:VTODO
DTSTAMP:20240313T101235Z
UID:b42fa285-90ce-4e4a-833c-1fd6779fe952
SEQUENCE:1
CREATED:20240304T092057Z
LAST-MODIFIED:20240313T101213Z
SUMMARY:Umzug von Google nach xyz
STATUS:NEEDS-ACTION
END:VTODO

BEGIN:VTODO
DTSTAMP:20240313T101122Z
UID:2962419142319295270
SEQUENCE:1
LAST-MODIFIED:20240313T101050Z
SUMMARY:Weitere fehlende Alben in Google Fotos finden und an Immich und S3 übertragen
PRIORITY:1
STATUS:NEEDS-ACTION
RELATED-TO;RELTYPE=PARENT:b42fa285-90ce-4e4a-833c-1fd6779fe952
END:VTODO

BEGIN:VTODO
UID:3097a87e-f7e3-44dd-81d6-30ce5be59255
SUMMARY:Nextcloud Bookmarks and Floccus
DESCRIPTION:https://www.reddit.com/r/selfhosted/comments/14k61n8/bookmarks_
 manager/?utm_source=share&utm_medium=web2x&context=3
RELATED-TO:b42fa285-90ce-4e4a-833c-1fd6779fe952
STATUS:COMPLETED
PERCENT-COMPLETE:100
COMPLETED;VALUE=DATE:20240314
PRIORITY:5
DTSTAMP:20240314T073712Z
END:VTODO
```

**Comment by alainm23** [comment]
I just released version 4.5.4, could I sync again and check if the problem persists?

**Comment by oktayacikalin** [comment]
I had to update, disconnect and reconnect to my Nextcloud instance, in order to show it up correctly. Looks good now :+1:
